.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.0.0-rc.1

```

Comment:
You can use the official release now instead of the rc1:
https://github.com/helm/chart-releaser-action/releases

Reply:
```suggestion
        uses: helm/chart-releaser-action@v1.0.0

```